### PR TITLE
Fixup: Add performer from search

### PR DESCRIPTION
**Pull request by Tom-Estes**
#### Database Migration
NO

#### Description
Add performer from search.

### CODE REVIEW

File: frontend/src/Store/Actions/addMovieActions.js [hunk 1]
```diff
@@ -28,17 +28,25 @@ export const defaultState = {
...
+    monitor: 'movieOnly',
+    qualityProfileId: 0,
+    searchForPerformer: true,
+    tags: []
   }
 };
 
 export const persistState = [
-  'addMovie.defaults'
+  'addMovie.defaults',
```
Comment (Qstick) [inline frontend/src/Store/Actions/addMovieActions.js]: addMovie.movieDefaults

File: frontend/src/Store/Actions/addMovieActions.js [hunk 2]
```diff
@@ -28,17 +28,25 @@ export const defaultState = {
...
+  'addPerformer.defaults'
```
Comment (Qstick) [inline frontend/src/Store/Actions/addMovieActions.js]: addMovie.performerDefaults

**Review by Qstick — CHANGES_REQUESTED** [review]

File: frontend/src/Store/Actions/addMovieActions.js (hunk 1 above)
Comment (Tom-Estes) [reply frontend/src/Store/Actions/addMovieActions.js]: Fixed

File: frontend/src/Store/Actions/addMovieActions.js (hunk 2 above)
Comment (Tom-Estes) [reply frontend/src/Store/Actions/addMovieActions.js]: Fixed

**Pull request merged**